p5js/FaceApi/FaceApi_Image_Landmarks/sketch.js

```diff
@@ -0,0 +1,116 @@
+let faceapi;
+let img;
+let detections;
+
+// by default all options are set to true
+const detection_options = {
+    withFaceLandmarks: true,
+    withFaceExpressions: false,
+    withFaceDescriptors: false,
+}
+
+function preload(){
+    img = loadImage('assets/frida.jpg')
+}
+
+function setup() {
+    createCanvas(200, 200);
+    img.resize(width, height);
+
+    faceapi = ml5.faceApi(detection_options, modelReady)
+    textAlign(RIGHT);
+}
+
+function modelReady() {
+    console.log('ready!')
+    console.log(faceapi)
+    faceapi.detectSingle(img, gotResults)
+
+}
+
+function gotResults(err, result) {
+    if (err) {
+        console.log(err)
+        return
+    }
+    // console.log(result)
+    detections = result;
+
+    // background(220);
+    background(255);
+    image(img, 0,0, width, height)
+    if (detections) {
+        // console.log(detections)
+        drawBox(detections)
```

Comment:
This is great, but similarly to PoseNet we might also consider examples that show using smaller bits of the data?

Reply:
@shiffman - Oh I'm not sure I'm familiar. Do you mean that PoseNet is compressing the images?

Reply:
Sorry, I was not clear at all! Let me try again! When we first introduced PoseNet the example that draws all of the keypoints and all of the skeleton examples overwhelmed users so it was nice to have one that, say, just shows you how to draw a circle on the nose or both eyes. Is there something similar we could do here? Like a simpler example that just draws the mouth? And maybe calculates the distance between top and bottom of mouth and maps that to a color or sound?

Reply:
@shiffman - ah yes, I think this is what we were mentioning in https://github.com/ml5js/ml5-examples/pull/161#discussion_r291630778